docs/modules/ROOT/pages/reference/ci-cd.adoc

```diff
@@ -0,0 +1,31 @@
+= AppCat CI/CD Processes
+
+AppCat contains many repositories.
+They all converge in the `component-appcat`, which is responsible for deploying everything.
+
+In order to streamline our development process, we improved the CI/CD with following steps:
+
+* Creating a PR in an AppCat repository automatically builds images and packes
+* Creating a PR in an AppCat repository automatically creates a PR on the component that reflect the changes, so it can instantly be deployed any SYN cluster to test
```

Comment:
```suggestion
* Creating a PR in an AppCat repository automatically creates a PR on the component that reflects the changes, so it can instantly be deployed any SYN cluster to test
```